controller/gui/style.css

```diff
@@ -87,7 +91,8 @@
   color: black;
   line-height: 1;
   display: grid;
-  grid-template-columns: 10fr 1fr 1fr;
+  grid-template-columns: 10fr auto auto auto;
+  grid-gap: 1rem;
```

Comment:
I'd use `3rem` to give more breathing room to the elements.

Reply:
Right, looks nicer: https://github.com/dividat/playos/pull/70/commits/bcd92f83a4cff0b5d47c89b71ef4b5c0369d52a3